internal/backend/runtime/omni/migration/migrations.go

```diff
@@ -1918,3 +1918,28 @@ func populateNodeUniqueTokens(ctx context.Context, st state.State, _ *zap.Logger
 
 	return err
 }
+
+func moveClusterTaintFromResourceToLabel(ctx context.Context, st state.State, _ *zap.Logger, _ migrationContext) error {
+	clusterTaints, err := safe.ReaderListAll[*omni.ClusterTaint](ctx, st)
+	if err != nil {
+		return err
+	}
+
+	for taint := range clusterTaints.All() {
+		if err = safe.StateModify(ctx, st, omni.NewClusterStatus(resources.DefaultNamespace, taint.Metadata().ID()),
+			func(res *omni.ClusterStatus) error {
+				res.Metadata().Labels().Set(omni.LabelClusterTaintedByBreakGlass, "")
+
+				return nil
+			},
+		); err != nil {
```

Comment:
Might need to add `WithPhase` any here to the modify call, otherwise clusters stuck in destroying will make this migration fail.